server/index.js

```diff
@@ -1,25 +1,28 @@
-/* eslint-disable no-console */
 const path = require('path');
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
-const Restaurant = require('../database/Restaurant.js');
+const cors = require('cors');
+const db = require('../database/index');
+const { searchById } = require('../database/Restaurant');
 
 const app = express();
-
-app.use(express.static(path.join(__dirname, '../client/dist')));
-
+app.use(cors());
 app.use(morgan('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(
+  '/r/:restaurants',
```

Comment:
Try making this route on line 14 and the one on line 19 more distinct from each other since they have different purposes.

Reply:
Updated to reflect /api/restaurant - will test if it works